notebook/utils/data_loader.py

```diff
@@ -24,11 +27,18 @@ def __init__(self, train_val_reader=None, test_reader=None, validation_split=0.2
         """
 
         if train_val_reader is not None:
-            indices = list(range(len(train_val_reader)))
-            np.random.RandomState(0).shuffle(indices)
-            split = int(np.floor(validation_split * len(train_val_reader)))
-            print("Split: ", split)
-            train_indices, val_indices = indices[split:], indices[:split]
+            if split_by_latitude:
+                lat = train_val_reader.labels.geometry.centroid.apply(lambda p: p.y)
+                train_indices = (lat[lat <= lat.quantile(1 - validation_split)]).index.to_list()
```

Comment:
lat.index uses loc-style, but sentinel_1_reader.py accesses data using iloc-style method.

Reply:
What's the easiest fix in your eyes?

Reply:
Update original labels index, or do something here?

Reply:
So here's the pros and cons.  if you fix this, it will be more time consuming (as opposed to only replacing loc with iloc in the other file), but it will not be affecting the author-created files.  if you fix the other file, it will be easier, but it will be modifying the author's original file.   

With that being said, to answer your question, fixing the other file is "easier".